composer.json

```diff
@@ -23,7 +23,8 @@
     "support": {
         "issues": "https://github.com/sulu/skeleton/issues",
         "source": "https://github.com/sulu/skeleton",
-        "docs": "http://docs.sulu.io/"
+        "docs": "http://docs.sulu.io/",
+        "chat": "https://sulu.io/services-and-support#chat"
```

Comment:
had some strange cache issue on the windows C and needed to invalidate the cache so added the `chat` url: https://getcomposer.org/doc/04-schema.md#support